Sources/MapboxCoreNavigation/BillingHandler.swift

```diff
@@ -76,48 +71,48 @@ private final class ProductionBillingService: BillingService {
     }
 
     func getSKUTokenIfValid(for sessionType: BillingHandler.SessionType) -> String {
-        MapboxCommon_Private.BillingService.getSessionSKUTokenIfValid(for: tripSku(for: sessionType))
+        BillingServiceNative.getInstance().getSessionSKUTokenIfValid(for: tripSku(for: sessionType))
```

Comment:
nit: I would extract `BillingServiceNative.getInstance()` into a computed property. 

```swift
var nativeBillingService: <WHAT TYPE HERE?> {
    BillingServiceNative.getInstance()
}
```

Reply:
Nit: name it `shared`. 🙂

Reply:
Introduced `NativeBillingService.shared` for this purpose.

Reply:
I did some tests and see that `BillingServiceNative.getInstance()` somehow caches its instances (they're sometimes different, but still reused). Hopefully using only one shared instance won't cause any problems.